itest/send_test.go

```diff
@@ -65,6 +65,7 @@ func testBasicSendUnidirectional(t *harnessTest) {
 					ev.SendState)
 
 				return ev.SendState == broadcastState
+			default:
```

Comment:
Should these default cases trigger a test failure? Sounds like there shouldn't be any other events here?

Reply:
Good point, I think type switches are a special case where the `singleCaseSwitch` lint is not triggered, and a type assertion is not suggested.

For these types, a fitting type assertion is actually generated from proto compilation; switched to those.